language/benchmarks/src/bench.move

```diff
@@ -0,0 +1,73 @@
+// `cargo bench` will call a public function in this module.
+// The Module must be called `Bench` and the set of public functions are callable from the bench (Rust code).
+// `benches/transaction.rs` contains the calling code.
+// The idea is that you build your scenario with a public entry point and a bunch of private functions as needed.
+module Bench {
+
+    //
+    // Global helpers
+    //
+    assert(check: bool, code: u64) {
+        if (check) () else abort code
+    }
+
+    //
+    // `arith` benchmark
+    //
+    public arith() {
+        let i = 0;
+        // 10000 is the number of loops to make the benchmark run for a couple of minutes, which is an eternity.
```

Comment:
Could these be controllable from the outside, as a benchmark parameter?

Reply:
we could, though not sure there is any value if that touches the Rust code.
In here that is easy to change and it was just to give a sense to people playing with this.
To be clear also I may have given the wrong reference here, the code runs in a short time but the benchmark takes 2/3 minutes. 
That part is controllable from the outside (I suppose running or configuring the setup) but this was just doing enough operations that are meaningful to a benchmark